_backend/Lib/L10n.php

```diff
@@ -116,6 +116,7 @@ public static function languageDirectory($lang)
         'ko' => '한국어',
         'lt' => 'Lietuvių kalba',
         'ms' => 'bahasa Melayu',
+        'mr' => 'मराठी'
```

Comment:
I suppose you'll need to add `,` at the end of this line like other lines.

Reply:
Yes, because otherwise it would break the code.